core/net/rpl/rpl-mrhof.c

```diff
@@ -118,6 +120,23 @@ reset(rpl_dag_t *dag)
 }
 
 static void
+dao_ack_callback(rpl_parent_t *p, int status)
+{
+  if(status == RPL_DAO_ACK_UNABLE_TO_ACCEPT_ROOT) {
```

Comment:
Why treat this case differently? Shouldn't we try another parent if the root doesn't want to take our DAO? Would seem to make sense to me if the problem at the root is a neighbor table full, but space left for routing entries.